Code/Editor/ViewPane.cpp

```diff
@@ -787,10 +787,7 @@ void CLayoutViewPane::SetViewportFOV(const float fovDegrees)
         pRenderViewport->SetFOV(fovRadians);
 
         // if viewport camera is active, make selected fov new default
-        if (pRenderViewport->GetViewManager()->GetCameraObjectId() == GUID_NULL)
-        {
-            SandboxEditor::SetCameraDefaultFovRadians(fovRadians);
-        }
```

Comment:
this looks like something to investigate further looks like this is done to prevent the setting from getting saved to settings registry. if its always null them maestro might just save it anyways. might add a breakpoint here and mess with the fov and see if this logic gets called.